lib/src/settings/mobile_position_settings.dart

```diff
@@ -0,0 +1,17 @@
+class MobilePositionSettings {
```

Comment:
Can't we use 'EdgeInsets' class instead of this class?

Reply:
Its about position in stack, there is natural values, like top: 12, not top: EdgeInsets, more context friendly.
Also, it will cause to use edgeInsets.left, edgeInsets.right in code. (Not so pleasant).
Also, it will shadow, that we can control logic like  top/bottom - appearance/disappearance. (per strategy, that may changed in setting of end user).
Also, Mobile in naming, it means, there is must be Desktop settings that totally different and must not be mixed together.
Edge insets will cause really messy mix.

Reply:
Fine, I see, It's okay then